gui/gm-editor.lua

```diff
@@ -339,7 +339,7 @@ function GmEditorUi:editSelected(index,choice,opts)
         if self:getSelectedEnumType() and not opts.raw then
             self:editSelectedEnum()
         elseif trg_type=='number' or trg_type=='string' then --ugly TODO: add metatable get selected
-            dialog.showInputPrompt(tostring(trg_key),"Enter new value:",COLOR_WHITE,
+            dialog.showInputPrompt(tostring(trg_key),"Enter new "..tostring(trg.target:_field(trg_key)._type).." value:",COLOR_WHITE,
```

Comment:
This one broke editing for primitive fields in Lua tables - e.g. in `gui/gm-editor df.global`, trying to edit a number triggers an error, because df.global._field isn't defined.